packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -68,22 +70,34 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
     gridTiles.push(localVideoComponent);
   }
 
+  const horizontalGallery = useMemo(() => {
+    if (horizontalGalleryTiles.length === 0) {
+      return null;
+    }
+    /* @conditional-compile-remove(pinned-participants) */
+    if (isNarrow) {
+      return <ScrollableHorizontalGallery horizontalGalleryElements={horizontalGalleryTiles} />;
+    }
+    return (
+      <VideoGalleryResponsiveHorizontalGallery
+        isNarrow={isNarrow}
+        shouldFloatLocalVideo={true}
+        horizontalGalleryElements={horizontalGalleryTiles}
+        styles={styles?.horizontalGallery}
+      />
+    );
+  }, [isNarrow, horizontalGalleryTiles, styles?.horizontalGallery]);
+
   return (
-    <Stack horizontal={false} styles={rootLayoutStyle}>
+    <Stack horizontal={false} styles={rootLayoutStyle} tokens={{ childrenGap: '0.5rem' }}>
```

Comment:
Anywhere using `tokens={{ childrenGap: '0.5rem' }}`, pull the object into a const defined outside the render body (or useMemo it) to prevent this being a new object on every render
```suggestion
    <Stack horizontal={false} styles={rootLayoutStyle} tokens={tileGap}>
```